katello/katello-backup

```diff
@@ -47,6 +55,64 @@ optparse = OptionParser.new do |opts|
   @dir = ARGV[0].dup
 end
 
+@databases.delete 'postgres' if @is_foreman_proxy_content
+
+def configure_configs
+  base_configs=     [
+                      "/etc/foreman-proxy",
+                      "/etc/httpd",
+                      "/etc/foreman-installer",
+                      "/etc/pki/katello",
+                      "/etc/pki/katello-certs-tools",
+                      "/etc/pki/pulp",
+                      "/etc/pulp",
+                      "/etc/puppet",
+                      "/etc/qpid",
+                      "/etc/qpid-dispatch",
+                      "/root/ssl-build",
+                      "/var/www/html/pub",
+                      "/etc/squid",
+                      '/opt/puppetlabs/puppet/cache/foreman_cache_data',
+                      '/opt/puppetlabs/puppet/ssl/',
+                      '/var/lib/puppet/foreman_cache_data',
+                      '/var/lib/puppet/ssl',
+                      "/var/lib/tftpboot",
+                      "/etc/dhcp",
+                      "/var/lib/dhcpd",
+                      "/etc/named*",
+                      "/var/named",
+                      "/usr/share/xml/scap"
+
+                    ]
+  katello_configs = [
+                      "/etc/candlepin",
+                      "/etc/foreman",
+                      "/etc/hammer",
+                      "/etc/puppetlabs",
+                      "/etc/sysconfig/tomcat*",
+                      "/etc/tomcat*",
+                      "/var/lib/candlepin",
+                    ]
+  if @is_foreman_proxy_content
+    capsule_certs_tar = `cat /etc/foreman-installer/scenarios.d/foreman-proxy-content-answers.yaml | grep certs_tar | awk '{print $2}'`.chomp
+    unless File.exist?(capsule_certs_tar) == true
+       puts "Capsule certs tar file is not present on the system in path #{capsule_certs_tar} please move the file back to that location or generate a new one on the main server."
+       exit(-1)
+    end
+    backup_configs = base_configs.push(capsule_certs_tar)
+  else
+    backup_configs = base_configs + katello_configs
+  end
+  @options[:features].map(&:downcase)
+  feature_configs = []
+  feature_configs.push("/var/lib/tftpboot") if @options[:features].include?("tftp") || @options[:features].include?("all")
+  feature_configs += ["/var/named/", "/etc/named*"] if @options[:features].include?("dns") || @options[:features].include?("all")
+  feature_configs += ["/var/lib/dhcpd", "/etc/dhcp"] if @options[:features].include?("dhcp") || @options[:features].include?("all")
+  feature_configs.push("/usr/share/xml/scap") if @options[:features].include?("openscap") || @options[:features].include?("all")
```

Comment:
You could DRY up these couple lines by making a utility method to check whats included in `options[:features]`